src/OrganisationComp.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import './App.css';
+
+
+
+class OrganisationList extends Component {
+  renderOrganisation= () => {
+    const orgData=this.state.orgData;
+      return orgData.map(function renderData(detailsOfServiceProviders) {
+        return (
+            <Details
```

Comment:
Great, but where is this Details component? you are not importing it here, and it is not included in the pull request.